features/Asset.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Playwright HRMS Asset
+Feature Description : To run Login tests with playwright library
+
+@Asset
+Scenario: Verfiy HRMS Create Asset
+Given Login to HRMS
+Then Create Category
+Then Create Subcategory
+Then Create Item
```

Comment:
Use proper step names ex: user creates item (if you passing any parameters add in the step instead of hard coding inside the step)

Reply:
Changed proper step names and removed Hard code